app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -52,8 +55,10 @@ import javax.inject.Singleton
 @RunWith(AndroidJUnit4::class)
 class AdministratorControlsActivityTest {
 
-  @Inject lateinit var profileTestHelper: ProfileTestHelper
-  @Inject lateinit var context: Context
+  @Inject
+  lateinit var profileTestHelper: ProfileTestHelper
```

Comment:
Single line.
`@Inject lateinit var profileTestHelper: ProfileTestHelper`

Reply:
Actually you dont need thi profileTetsHelper

Reply:
ProfileTestHelper is needed because without it the test-cases will fail as profiles have not been  initialized before the test-cases run